<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsultProductDialogClass</class>
 <widget class="QDialog" name="ConsultProductDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar Producto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Código barra:</string>
       </property>
       <property name="buddy">
        <cstring>barCodeLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="barCodeLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="barCodePushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>C&amp;onsultar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Código:</string>
       </property>
       <property name="buddy">
        <cstring>idLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="idLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="idPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cons&amp;ultar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Buscar:</string>
       </property>
       <property name="buddy">
        <cstring>nameSearchProductLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SearchProductLineEdit" name="nameSearchProductLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWebView" name="webView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>100</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>SearchProductLineEdit</class>
   <extends>QLineEdit</extends>
   <header>search_product/search_product_line_edit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>barCodeLineEdit</tabstop>
  <tabstop>barCodePushButton</tabstop>
  <tabstop>idLineEdit</tabstop>
  <tabstop>idPushButton</tabstop>
  <tabstop>nameSearchProductLineEdit</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsultProductDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barCodeLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>barCodePushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>idPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
